server/cluster/metadata/cluster_metadata.go

```diff
@@ -251,6 +251,32 @@ func (c *ClusterMetadata) GetTable(schemaName, tableName string) (storage.Table,
 	return c.tableManager.GetTable(schemaName, tableName)
 }
 
+func (c *ClusterMetadata) CreateTableMetadata(ctx context.Context, request CreateTableMetadataRequest) (CreateTableMetadataResult, error) {
+	log.Info("create table start", zap.String("cluster", c.Name()), zap.String("schemaName", request.SchemaName), zap.String("tableName", request.TableName))
+
+	_, exists, err := c.tableManager.GetTable(request.SchemaName, request.TableName)
+	if err != nil {
+		return CreateTableMetadataResult{}, err
+	}
+
+	if exists {
+		return CreateTableMetadataResult{}, ErrTableAlreadyExists
+	}
+
+	// Create table in table manager.
+	table, err := c.tableManager.CreateTable(ctx, request.SchemaName, request.TableName, request.PartitionInfo)
+	if err != nil {
+		return CreateTableMetadataResult{}, errors.WithMessage(err, "table manager create table")
+	}
+
+	ret := CreateTableMetadataResult{
+		Table: table,
+	}
+
+	log.Info("create table metadata succeed", zap.String("cluster", c.Name()), zap.String("result", fmt.Sprintf("%+v", ret)))
```

Comment:
For the struct, will `zap.Object` work?

Reply:
`zap.object` is used to print objects, but it needs the type to be output to implement a serialized interface, which seems a bit troublesome, and I observed that PD also uses this method to print Objects instead of `zap.object`.

Reply:
If we use `zap.object`, we need to implement an interface for each struct like this:
```
type Foo struct {
  name string
  bar  Bar
}

func (f Foo) MarshalLogObject(enc zapcore.ObjectEncoder) error {
  enc.AddString("name", f.name)
  
  // Assuming bar is an object:
  if err := enc.AddObject("bar", f.bar); err != nil {
    return err
  }
}
```